src/soundio/sounddeviceportaudio.cpp

```diff
@@ -820,7 +856,9 @@ int SoundDevicePortAudio::callbackProcessDrift(
             //qDebug() << "callbackProcess read:" << (float)readAvailable / outChunkSize << "Buffer empty";
         }
     }
-    return paContinue;
+    return m_streamState.load(std::memory_order_acquire) == StreamState::STOP
```

Comment:
This is not used for syncing. Does relax also work? We may avoid the condition to store the return value directly.

Reply:
Yeah, I initially stored the return value in an atomic, but IIRC I needed the extra state when starting the streams. But I don't remember so I will revise this.